share/rpcauth/rpcauth.py

```diff
@@ -38,9 +39,14 @@ def main():
     salt = generate_salt(16)
     password_hmac = password_to_hmac(salt, args.password)
 
-    print('String to be appended to bitcoin.conf:')
-    print('rpcauth={0}:{1}${2}'.format(args.username, salt, password_hmac))
-    print('Your password:\n{0}'.format(args.password))
+    if args.output:
+        file = open(args.output, "x")
```

Comment:
In commit "rpcauth: Support storing credentials in a file" (ff5d7fa1e4c340657110436a29aecff85de6eb80)

Doesn't really matter but 

```python
with open(args.output, "x") as file:
   file.write(...)
```

Is slightly better style because it ensures that the file is closed immediately after the write.

Also may wish to avoid `file` variable name and use something like `f` or `fp` since `file` is a builtin function.